src/GrpcProtos/capture.proto

```diff
@@ -323,13 +323,18 @@ message SystemMemoryUsage {
   int64 available_kb = 4;
   int64 buffers_kb = 5;
   int64 cached_kb = 6;
+  // These fields are retrieved from /proc/vmstat.
+  int64 pgfault = 7;
```

Comment:
shouldn't we include minor in the name?

minorpgfault and majpgfault?

Reply:
Thanks for the suggestion!

For these fields, we are using the exact same names as in the file `memory.stat`. The `memory.stat.pgmajfault` reports the number of major page faults but the `memory.stat.pgfault` reports the total number of page faults (rather than the number of minor page faults).

I would like to extract the raw value in the file and do the conversion to minor page faults when we need to use these values. What do you think?

Reply:
Yes, of course. Agree 100%